java-rest-service/accelerator.yaml

```diff
@@ -83,7 +83,37 @@ accelerator:
   imports:
   - name: java-version
   - name: spring-boot-postgresql
+    expose:
+      - name: databaseName
```

Comment:
Could argue that those two don't need dependsOn as they happen to be common to both fragments.